aptos-move/aptos-gas-schedule/src/ver.rs

```diff
@@ -76,4 +78,5 @@ pub mod gas_feature_versions {
     pub const RELEASE_V1_13: u64 = 18;
     pub const RELEASE_V1_14: u64 = 19;
     pub const RELEASE_V1_15: u64 = 20;
+    pub const RELEASE_V1_16: u64 = 21;
```

Comment:
could you please add a couple more numbers while you at it? thank you!

Reply:
For previous releases?

Reply:
for future releases :D

Reply:
done